distribution/lib/Standard/Table/0.0.0-dev/src/Column.enso

```diff
@@ -2140,13 +2141,11 @@ type Column
 
        Arguments:
        - max_rows: specifies the maximum number of rows to read.
-         If `Nothing`, all available rows are returned.
-       - warn_if_more_rows: if set to `True`, a warning is attached to the
-         result if the number of rows returned by the query exceeds `max_rows`.
-    read : (Nothing | Integer) -> Boolean -> Column
-    read self (max_rows : Nothing | Integer = Nothing) (warn_if_more_rows:Boolean = True) =
+    @max_rows Rows_To_Read.default_widget
+    read : Rows_To_Read -> Column
+    read self (max_rows : Rows_To_Read = ..All_Rows) =
```

Comment:
This `..All_Rows` looks so much clearer than `Nothing`. I'm really happy with our switching towards more meaningful arguments. And with autoscoping it is still quite terse :)